examples/platform/linux/BUILD.gn

```diff
@@ -138,6 +158,36 @@ source_set("app-main") {
     public_deps += [ jsoncpp_root ]
   }
 
+  if (chip_with_trusty_os) {
+    public_deps += [ "${chip_root}/third_party/libtrustymatter" ]
+  }
+
+  if (chip_with_imx_ele) {
+    public_deps +=
+        [ "${chip_root}/third_party/imx-secure-enclave:libelematter" ]
+  }
+
+  if (chip_with_trusty_os) {
+    defines += [ "CHIP_ATTESTATION_TRUSTY_OS=1" ]
+  } else {
+    defines += [ "CHIP_ATTESTATION_TRUSTY_OS=0" ]
+  }
+
+  if (chip_with_trusty_os) {
+    defines += [ "CHIP_OP_KEYSTORE_TRUSTY_OS=1" ]
+  } else {
+    defines += [ "CHIP_OP_KEYSTORE_TRUSTY_OS=0" ]
+  }
+
+  if (chip_with_imx_ele) {
+    defines += [ "CHIP_OP_KEYSTORE_ELE=1" ]
+    #TODO enable this when official tool ready
+    #   defines += [ "CHIP_ATTESTATION_ELE=1" ]
```

Comment:
If there is no official tool, we are unable to test these/maintain these. Please separate things out and prepare a patch after the official tool is ready